translations/messages-en.properties

```diff
@@ -976,3 +976,6 @@ contact.deceased.date.prefix = Died
 contact.deceased.view = View deceased ({{count}})
 user.password.current = Current password
 password.incorrect = Password is not correct.
+online.action.title = Please make sure that you are online.
+online.action.message = This action requires you to be online, please try again when you have network access.
+close= Close
```

Comment:
I don't think the `close` translation is used anywhere? If so, please delete it.